apps/next/components/Details/Info.tsx

```diff
@@ -10,13 +10,13 @@ const InfoLine = ({
   alternativeText,
 }: {
   leftText: string
-  rightText: string
-  alternativeText: string
+  rightText: string | undefined;
```

Comment:
Why would `rightText` ever be undefined? Wouldn't the default be `"Not Set"`?

Reply:
Same reason here, it's for loading animation purpose.